core/src/test/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -199,26 +49,13 @@ public void testIsTimeout() throws IOException {
     }
 
     public void testClusterHealth() throws IOException {
-        ShardCounter counter = new ShardCounter();
```

Comment:
To me this is not far away from testing trivial getters but I'll reintroduce the assertions.